docs/tutorial/fault_tolerations.md

```diff
@@ -1,346 +1,494 @@
-# worker和ps容错样例
+# Fault-tolerance and Elasticity Experiments of DLRover ElasticJob
 
-## worker容错示例
+The tutorial shows experiments to test the fault-tolerance and elasticity
+of DLRover ElasticJob. In the experiments, we use the chaos enginerring toolkit
+[chaosblade](https://github.com/chaosblade-io/chaosblade) to simulate fault scenarios.
 
-在任务运行过程中删除worker-i对应的pod，之后dlrover master会重新拉起一个pod。work-i对应的pod的名称会发生变化，新创建的pod的启动命令和被kill掉的pod的启动命令相同，启动后参与组网，并进行训练。期间其他worker不受影响。
+## Preliminary
 
-### 启动作业
+- Create a k8s cluster and configure cluster credentials on your local computer.
```

Comment:
why we need to configure the cluster credentials on the local machine?

Reply:
The  k8s cluster is usually a remote cluster and we need the credential to connect the cluster.